### ISSUE TITLE
(AWS::KMS::Alias) false-positive E3008 for `KeyId`

### ISSUE THREAD

**Issue by Tommes-M** [aws-cloudformation/cfn-lint#2952]
### CloudFormation Lint Version

v0.83.3

### What operating system are you using?

Windows 10

### Describe the bug

`AWS::KMS::Key` now supports `!GetAtt <ResourceName>.KeyId`, but `AWS::KMS::Alias` displays this error message when trying to set the property `TargetKeyId` for it:

```
[cfn-lint] E3008: Property "TargetKeyId" can Fn::GetAtt to a resource attribute "Arn" at Resources/<ResourceName>/Properties/TargetKeyId/Fn::GetAtt
```

### Expected behavior

`!GetAtt <ResourceName>.KeyId` displays no error message when trying to set it for the property `TargetKeyId`

### Reproduction template

```yaml
AWSTemplateFormatVersion: 2010-09-09

Resources:
  KMSKey:
    Type: AWS::KMS::Key

  KMSAlias:
    Type: AWS::KMS::Alias
    Properties:
      AliasName: <alias name>
      TargetKeyId: !GetAtt KMSKey.KeyId   <-- Here is the bug / error
